mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -45,90 +62,55 @@ var channelCmd = &cobra.Command{
 			channelName = channelSet
 		}
 
-		switch channelName {
+		channelNameSeperated := strings.Split(channelName, ":")
+
+		ContextContent, ok := mctlCfg.Contexts[focusedContext]
+		if !ok {
+			return errors.New("error while trying to fetch context content")
+		}
+
+		switch channelNameSeperated[0] {
 		case "stable":
-			if err := searchAndReplace(utils.DockerComposeFile, "edge", "stable"); err != nil {
-				log.Fatal("Error subscribing to release channel:", err)
-				return
+			ContextContent.Channel = channelNameSeperated[0]
+			if len(channelNameSeperated) == 1 {
+				ContextContent.Version = "latest"
+			} else {
+				ContextContent.Version = channelNameSeperated[1]
 			}
-			log.Info("Subscribed to stable channel.")
 
 		case "edge":
-			if err := searchAndReplace(utils.DockerComposeFile, "stable", "edge"); err != nil {
-				log.Fatal("Error subscribing to release channel:", err)
-				return
-			}
-
-			log.Info("Subscribed to edge channel.")
+			ContextContent.Channel = channelNameSeperated[0]
+			ContextContent.Version = "latest"
 
 		default:
-			currentChannel := getCurrentContextChannel()
+			currentChannel := mctlCfg.Contexts[focusedContext].Channel
```

Comment:
When users try to switch to something other than stable or edge, they should be shown a message indicating that these are the only two channels available.